### DISCUSSION TITLE
FMU-Python dependencies

### DISCUSSION THREAD

**Discussion by AishwaryaRaghavendra** [StephenSmith25/PythonFMU3#2] [Q&A]
Conside source system where the FMU was built from python script including all their dependencies which are thrid-party packages eg. numpy, pandas etc. using requirements.txt file.

Does this FMU when used in another destination system requires all these dependencies to be again installed in destination system in order to make this FMU run there?

**Comment by StephenSmith25** [comment]
Sorry, Just seeing this now.

In the executing environment you will require all of the dependencies to be installed. Internally, pythonfmu launches a python executable and executes your python script. If the python script imports numpy, scipy ect.. your importing environment will need to know where these are (as they are not shipped in the FMU).
